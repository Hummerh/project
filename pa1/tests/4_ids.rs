// Ids may contain letters (a-z, A-Z), digits, or underscores, but they must
// not start with a digit.
(abc, ABC, abc123, abc_123, _123abc_)
