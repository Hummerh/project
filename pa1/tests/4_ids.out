(
ID(abc)
,
ID(ABC)
,
ID(abc123)
,
ID(abc_123)
,
ID(_123abc_)
)
